masim.c

```diff
@@ -509,13 +513,14 @@ int parse_phase(char *lines[], int nr_lines, struct phase *p,
 		for (k = 0; k < nr_regions; k++) {
 			if (strcmp(fields[0], regions[k].name) == 0) {
 				a->mregion = &regions[k];
+                break;
```

Comment:
Why `break` here?

Reply:
A pattern corresponds to only one area of memory. So if match the corresponding region name, there's no need to check for others.

Reply:
Thanks for the clarification.  But this is a change that is not related with the 64bit fix, right?  Could you please separate this fix to another commit, and adjust the PR message?

Reply:
ok, sure